repose-aggregator/commons/utilities/src/main/java/org/openrepose/commons/utils/servlet/http/MutableHttpServletResponse.java

```diff
@@ -302,11 +302,16 @@ public void sendError(int code) throws IOException {
 
     @Override
     public void sendError(int code, String message) {
-        super.setStatus(code);
-        this.message = message;
+        setStatus(code, message);
         error = true;
     }
 
+    @Override
+    public void setStatus(int code, String message) {
+        super.setStatus(code, message);
+        this.message = message;
```

Comment:
But hopefully all this is moot and the new wrappers will be in place and we don't have to worry about it.